e2e/dractions/disapp.go

```diff
@@ -49,6 +53,10 @@ func EnableProtectionDiscoveredApps(ctx types.Context) error {
 		return err
 	}
 
+	if err := createNamespaces(ctx, appNamespace, log); err != nil {
+		return err
+	}
```

Comment:
We should delete the namespaces when undeploying the app, so we don't leave leftovers in users clusters in ramenctl.  `ramenctl test clean` assumes that running unprotect and undeploy will delete all namespaces created by the test.